docs/tutorials/multimodal/text_prediction/ner.md

```diff
@@ -45,6 +45,7 @@ model_path = f"./tmp/{uuid.uuid4().hex}-automm_ner"
 predictor = MultiModalPredictor(problem_type="ner", label=label_col, path=model_path)
 predictor.fit(
     train_data=train_data,
+    hyperparameters={'model.ner_text.checkpoint_name':'bert-base-cased'},
```

Comment:
Can we make this default? Not neccessary for 0.6.2 release, but better to refactor and make it simpler for users going forward (e.g. 0.7 release)

Reply:
It uses deberta base by default. I set it to bert-base-cased as it gives better demonstration results given such a short training time (300s) just for tutorial purpose.